src/qibolab/_core/instruments/qm/program/instructions.py

```diff
@@ -31,6 +31,14 @@ def _delay(pulse: Delay, element: str, parameters: Parameters):
             qua.wait(duration, element)
 
 
+def _virtualz(pulse: VirtualZ, element: str, parameters: Parameters):
+    if parameters.phase is not None:
+        phase = parameters.phase
+    else:
+        phase = normalize_phase(pulse.phase)
```

Comment:
```suggestion
    phase = parameters.phase if parameters.phase is not None else normalize_phase(pulse.phase)
```

Personal preference to avoid conditional assignment, when you just need conditional values.